src/app/page.tsx

```diff
@@ -127,8 +61,36 @@ export default function Home() {
     <main className={styles.main}>
       <section className={styles.todoSection}>
         <h1 className={styles.title}>Todo list</h1>
-        <AddTodoForm onAddTodo={handleAddTodo}  />
-        <TodoList data={todos} onTodoChange={handleTodoChange} onDeleteTodo={handleDeleteTodo}/>
+        <AddTodoForm onAddTodo={handleAddTodo} />
+        {loading ? (
+          <svg
```

Comment:
Could you create a Loader component and add this there so that this file is a bit cleaner?